.gitignore

```diff
@@ -2,3 +2,36 @@ node_modules
 package.json
 package-lock.json
 output
+
+### OCaml ###
+*.annot
+*.cmo
+*.cma
+*.cmi
+*.a
+*.o
+*.cmx
+*.cmxs
+*.cmxa
```

Comment:
None of these are necessary when using dune.

Reply:
This is the default generation from [gitignore.io](https://www.toptal.com/developers/gitignore/api/ocaml). Old habit :man_facepalming: